bin/opt8n/src/main.rs

```diff
@@ -1,102 +1,35 @@
+pub mod cmd;
```

Comment:
Is this PR supposed to be stacked ontop of https://github.com/ethereum-optimism/op-test-vectors/pull/65?

Reply:
Yes this is meant to be stacked on top of #65. I should have mentioned this in the description.